runtests.sh

```diff
@@ -66,6 +66,8 @@ export AWS_S3_REGION="not-set"
 prepare_venv
 pip3 install -r requirements.txt
 pip3 install -r tests/requirements.txt
+pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git@d7aaccf
```

Comment:
@yzainee for better maintainability, please set the commit hash in a variable in the script and use that variable with the `pip3 install` here.

Reply:
not using the commit hash as its expected to use the latest version comparator code everytime. let me know if you think otherwise

Reply:
@yzainee my comment was for `git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git`. Please keep the commit hash for utils in sync with the one used in the Dockerfile and also, although nitpick, please parameterize the commit hash for `fabric8-analytics-utils.git` in `runtests.sh`